schemas/rules/operators/numeric/sum.json

```diff
@@ -8,7 +8,7 @@
     "properties": {
       "+": {
         "type": "array",
-        "minItems": 2,
+        "minItems": 1,
```

Comment:
This probably means grand calculated summary allows one item now (as this operator rule is used by both GCS and CS)? Do we need to add extra python validation for GCS?

Reply:
Luckily its ok for GCS, if you look at `sum.json` the gcs values come from `sum_calculated_summary_values` which still has a minimum of 2